app/views/admin/move_advisers/confirm.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>Please confirm</h1>
+
+<p>Please confirm you want to:</p>
+<dl>
+  <dt>Move:</dt>
+  <% @form.advisers_to_move.each do |adviser| %>
+    <dd class="t-adviser">(<%= adviser.reference_number %>) <%= adviser.name %></dd>
+  <% end %>
+  <dt>From:</dt><dd><span class="t-from_firm">(<%= @form.from_firm.fca_number %>)
+    <%= @form.from_firm.registered_name %></span></dd>
+  <dt>To:</dt><dd><span class="t-to_firm">(<%= @form.to_firm.fca_number %>)
```

Comment:
There's no hard and fast rule on formatting test classes I think, but generally I'd expect to see hyphens than underscores. Nitpicky I know!